openmetadata-ui/src/main/resources/ui/src/components/common/rich-text-editor/RichTextEditorPreviewer.test.tsx

```diff
@@ -344,4 +350,89 @@ describe('Test RichTextEditor Previewer Component', () => {
 
     expect(markdownParser).toBeInTheDocument();
   });
+
+  it('Should render read more button if enableSeeMoreVariant is true and max length is less than content length', () => {
```

Comment:
Let's separate both cases
1. should render "more" if enableSeeMoreVariant is true
2. should render "more" if max length is less than content length

Also more inclined to follow above format to mention use-cases for more readable manner

Change "Read more" --> "More"
"Read less" --> "Less"

Reply:
Sure, I will change the use case text, however, we have separate cases if you look at the tests after that test.